src/material.js

```diff
@@ -1,7 +1,9 @@
-import shaderSources from './gl/shader_sources'; // built-in shaders
 import GLSL from './gl/glsl';
 import {StyleParser} from './styles/style_parser';
 
+let fs = require('fs');
```

Comment:
I originally tried this, and after some digging I found there is a problem: https://github.com/babel/babelify/issues/81

It looks like for now, we will need to keep this with a require statement, though I agree it's ugly